src/EpollWrapper.cpp

```diff
@@ -60,8 +60,9 @@ int EpollWrapper::waitForEvents()
 		throw std::runtime_error("epoll_wait:" + std::string(strerror(errno)));
 	}
 
-	if (nfds == 0)
+	if (nfds == 0) {
```

Comment:
The brackets are superfluous

Reply:
We get a "Add explicit braces to avoid dangling else" error without them.

I think it's because of the macro expansion. After the change the code expands to
```
if (nfds == 0)
	if (0) std::cerr << "epoll_wait: Timeout";
else
	if (0) std::cerr << "epoll_wait: " << nfds << " events";
``` 
After the No Ops are removed we have
```
if (nfds == 0)
else
``` 
which has a dangling else. 
With brackets we have
```
if (nfds == 0) {
}
else
``` 
Which is valid. I think the if check will be removed anyway. But it still throws an error